internal/wiz/structs.go

```diff
@@ -244,6 +244,31 @@ type SAMLGroupMappingCreateInput struct {
 	Projects        []string `json:"projects"`
 }
 
+// UpdateSAMLGroupMappingInput struct
+type UpdateSAMLGroupMappingInput struct {
+	ID    string                      `json:"id"`
+	Patch ModifySAMLGroupMappingPatch `json:"patch"`
+}
+
+// SAMLGroupMappingDetailsInput struct
+type SAMLGroupMappingDetailsInput struct {
```

Comment:
rename to `SAMLGroupDetailsInput`, add a struct comment to say it is incomplete that 'description' is missing as defined in schema